python/glow/gwas/log_reg.py

```diff
@@ -77,12 +81,15 @@ def logistic_regression(
     gwas_fx._validate_covariates_and_phenotypes(covariate_df, phenotype_df, is_binary=True)
     sql_type = gwas_fx._regression_sql_type(dt)
     genotype_df = gwas_fx._prepare_genotype_df(genotype_df, values_column, sql_type)
-    result_fields = [
-        # TODO: Probably want to put effect size and stderr here for approx-firth
+    base_result_fields = [
         StructField('tvalue', sql_type),
         StructField('pvalue', sql_type),
         StructField('phenotype', StringType())
     ]
+    if correction == correction_approx_firth:
+        result_fields = [StructField('effect', sql_type), StructField('stderr', sql_type)] + base_result_fields
```

Comment:
I'm not sure if it's valuable to state the number of iterations, but it does make sense to clarify when the fit failed. I think it may make more sense to have a nested column with `beta`, `stderr`, and `converged` for Firth, which allows us to leave the uncorrected `tvalue` and `pvalue`.